samples/example_subplots.py

```diff
@@ -1,13 +1,16 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+import click
```

Comment:
http://click.pocoo.org/5/python3/
I think Python 3 support in click is enough for what I'm doing.

I didn't faced any problem with my version
```
$ python --version
Python 3.5.2 :: Anaconda 4.1.1 (x86_64)
```